### PR TITLE
CIV-10441 Limit number of characters entered on the text boxes are not working as expected

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-10441

### Change description ###

Limit number of characters entered on the text boxes are not working as expected

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ElifDenizEsea — APPROVED** [review]

**Review by manish14836 — DISMISSED** [review]

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -3038,6 +3038,10 @@
     },
     "TEXT_TOO_LONG": "Reason must be 500 characters or fewer",
     "TEXT_TOO_MANY": "You've entered too many characters",
+    "ADDRESS_LINE1_TOO_MANY": "Address line1 must no longer than 70 characters",
```
Comment (jarekPierchala) [inline src/main/modules/i18n/locales/en.json:3041]: Address line1 must no longer than 70 characters - isn't it missing "be"? as
Address line1 must **be** no longer than 70 characters?

**Review by jarekPierchala — CHANGES_REQUESTED** [review]

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (jeswanth-hmcts) [reply src/main/modules/i18n/locales/en.json]: Good spot 'be'  is missing  added 'be' in text

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -3038,6 +3038,10 @@
     },
     "TEXT_TOO_LONG": "Rhaid i'r rheswm fod yn 500 llythyren neu lai.",
     "TEXT_TOO_MANY": "Rydych wedi defnyddio gormod o nodau",
+    "FULL_NAME_TOO_LONG": "Full name must be no longer than 70 characters",
+    "JOB_TITLE_TOO_LONG": "Job title must be no longer that 70 characters",
```
Comment (neeta-hmcts) [inline src/main/modules/i18n/locales/cy.json:3042]: must be no longer **than** .. same for following statements

Comment (jeswanth-hmcts) [reply src/main/modules/i18n/locales/cy.json:3042]: image
it is done As per AC text

**Review by laarmada — CHANGES_REQUESTED** [review]
where are the unit test to cover all that validations?

**Comment by jeswanth-hmcts** [comment]
> where are the unit test to cover all that validations?
 

> where are the unit test to cover all that validations?

added unit test

**Review by laarmada — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (jarekPierchala) [reply src/main/modules/i18n/locales/cy.json:3042]: "that" is obviously a mistake. Correct it to **than** please. Ill speak to Kiera to update ACs

**Review by jarekPierchala — CHANGES_REQUESTED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Review by laarmada — DISMISSED** [review]

**Review by kalachandrasekar1 — DISMISSED** [review]

**Review by portizdev — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**